src/Response/Response.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace FurnitureStoreAPI\Response;
+class Response
+{
```

Comment:
See my comment above: replace these with a single method that takes the response code, message and data and returns the response